### PR TITLE
docs(ko): add ko docs to the official site

### PR DESCRIPTION
**Pull request by niceplugin**
We are merging the documents from the personal domain into the official site.
After the merge, we plan to deactivate the personal domain.
The document content has been reviewed up to version 1.3.1 (though there are still some awkward translations ^^;).

### CODE REVIEW

**Comment by brc-dd** [comment]
Can you update docs/lunaria.config.json too?

**Comment by niceplugin** [comment]
I added 'ko'. Please check.

**Review by brc-dd — APPROVED** [review]
LGTM. Is it ready to merge?

File: docs/.vitepress/config/ko.ts [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+            { text: '검색', link: 'search' },
+            { text: '카본 광고', link: 'carbon-ads' }
+          ]
+        }
+      ]
+    }
+  ]
+}
+
+export const search: DefaultTheme.AlgoliaSearchOptions['locales'] = {
```
Comment (brc-dd) [inline docs/.vitepress/config/ko.ts:210]: ah this should be imported in docs/.vitepress/config/shared.ts

File: docs/ko/index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+titleTemplate: Vite & Vue로 구동되는 정적 사이트 생성기
+
+hero:
+  name: VitePress
+  text: Vite & Vue로 구동되는 정적 사이트 생성기
+  tagline: 마크다운으로 멋진 문서를 몇 분 안에
+  actions:
+    - theme: brand
+      text: VitePress란 무엇인가?
+      link: /guide/what-is-vitepress
```
Comment (brc-dd) [inline docs/ko/index.md:14]: adjust these links to /ko/

**Comment by niceplugin** [comment]
I have checked and corrected it.

**Pull request merged**